{"name": "Login with wrong email", "status": "passed", "description": "Signing in with the following credentials email : kamelmohsenkamel & password: Kimo2010", "attachments": [{"name": "SUCCESS: Login failed with incorrect credentials", "source": "9f4bd8ed-cfb0-4abc-a70b-d0f4bf22d2f7-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "5074d9c0-45f9-4131-93d4-68f9822e0eff-attachment.txt", "type": "text/plain"}], "start": 1586148102099, "stop": 1586148109798, "uuid": "900117e0-daba-4699-8e8b-2941a579fa81", "historyId": "05faaa24a4f795aa6d491dda8fe808bc", "testCaseId": "83e4f7dab5663a2784096447330dbf3e", "fullName": "Web_Testing.Tests.test_login.TestSignin#test_case_5", "labels": [{"name": "suite", "value": "Login Tests"}, {"name": "feature", "value": "Log in Tests"}, {"name": "severity", "value": "blocker"}, {"name": "story", "value": "Failing login"}, {"name": "subSuite", "value": "Login with wrong email"}, {"name": "parentSuite", "value": "End to End testing"}, {"name": "tag", "value": "Do"}, {"name": "tag", "value": "Login"}, {"name": "host", "value": "LAPTOP-8H6HECEC"}, {"name": "thread", "value": "19944-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Web_Testing.Tests.test_login"}]}